.github/workflows/pr-actions.yml

```diff
@@ -0,0 +1,124 @@
+name: prnet-actions
+on:
+  pull_request:
+    types: [opened]
```

Comment:
add `closed` somehow

Reply:
it will need another workflow, perhaps we can merge this PR start working with the events we have and later i can add the events that are not related to issue_comments that require a different workflow (different file as well) so we can move faster. up to you.